go-controller/pkg/kube/kube.go

```diff
@@ -76,6 +78,7 @@ type KubeOVN struct {
 	ANPClient            anpclientset.Interface
 	EIPClient            egressipclientset.Interface
 	EgressFirewallClient egressfirewallclientset.Interface
+	NetworkClient        ocpnetworkclientset.Interface
```

Comment:
maybe rename to `OCPNetworkClient`? `Network` is a bit overloaded in this repo :)